llabake/Hello-Books
# Feature(404 page) Implement 404 Page

#### What does this PR do?
This PR adds 404 page component

#### Description of Task to be completed?
creates the 404 Page
update book reducer with resource not found variable

#### How should this be manually tested?
clone the repo
cd into the project directory
run `npm install` to install all the dependency
run `npm run start-dev` to start the back-end server
run `npm run dev` to start the front-end server
then navigate to a broken URL or a resource that does not exist
you should see the 404 page loaded 
#### What are the relevant pivotal tracker stories?
157527617

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1440" alt="screen shot 2018-05-15 at 12 01 43 pm" src="https://user-images.githubusercontent.com/29184825/40053201-5210c3d2-5838-11e8-9a88-5f54d6e5ddc2.png">
<img width="1440" alt="screen shot 2018-05-15 at 12 17 34 pm" src="https://user-images.githubusercontent.com/29184825/40053651-fcdbce28-5839-11e8-9462-9f27dcc0fee9.png">
